tests/test_store.py

```diff
@@ -214,6 +220,35 @@ def test_view_query(
         # tests that the query was valid and successful, that a view subset has actually been retrived
         assert synapse_store_special_scope.storageFileviewTable.empty is False
 
+    @pytest.mark.parametrize(
+        "asset_view,columns,expectation",
+        [
+            ("syn62339865", ["path"], pytest.raises(ValueError)),
+            ("syn62340177", ["id"], pytest.raises(ValueError)),
+            ("syn23643253", ["id", "path"], does_not_raise()),
+        ],
+    )
+    def test_view_query_exception(
+        self,
+        synapse_store_special_scope: SynapseStorage,
+        asset_view: str,
+        columns: list,
+        expectation: str,
+    ) -> None:
+        project_scope = ["syn23643250"]
+
+        # set to use appropriate test view and set scope
+        CONFIG.synapse_master_fileview_id = asset_view
+        synapse_store_special_scope.project_scope = project_scope
+
+        # ensure approriate exception is raised
+        try:
+            with expectation:
+                synapse_store_special_scope.query_fileview(columns)
+        finally:
+            # reset config to default fileview
+            CONFIG.synapse_master_fileview_id = "syn23643253"
```

Comment:
Nit: Why is it important to reset the config to default fileview?  That, in my opinion, is more important to write as a comment, because we can read that you're resetting the `synapse_master_fileview_id` back to another synapse id.

Reply:
@thomasyu888 that makes sense, I can explain that more clearly